ui-library-filesharing-ui-components/api/UploadFileToAzureBlobStore/index.ts

```diff
@@ -0,0 +1,90 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+import { AzureFunction, Context, HttpRequest } from "@azure/functions";
+import * as multipart from "parse-multipart";
+import HTTP_CODES from "http-status-enum";
+
+const azureStorageConnectionString =
+  process.env["azureStorageConnectionString"];
+let storageAccountName;
+try {
+  storageAccountName =
+    azureStorageConnectionString.match(/AccountName=([^;]+)/)[1];
+} catch {
+  console.error("Invalid azure storage connection string.");
+}
+
+// Azure storage container name to be used for uploading files.
+// Must match the value in function.json `path`
+const fileSharingUploadsContainerName = "uploads";
+
+const httpTrigger: AzureFunction = async function (
+  context: Context,
+  req: HttpRequest
+): Promise<void> {
+  context.log(
+    "HTTP trigger function processed a request to upload a file to azure blob store."
+  );
+
+  if (!req.body || !req.body.length) {
+    context.res.body = { error: `Request body is not defined` };
+    context.res.status = HTTP_CODES.BAD_REQUEST;
+    return;
+  }
+
+  if (!req.query?.filename) {
+    context.res.body = { error: `filename is not defined` };
+    context.res.status = HTTP_CODES.BAD_REQUEST;
+    return;
+  }
+
+  if (!req.headers || !req.headers["content-type"]) {
+    context.res.body = {
+      error: `Content type is not sent in header 'content-type'`,
+    };
+    context.res.status = HTTP_CODES.BAD_REQUEST;
+    return;
+  }
+
+  context.log(
+    `*** Uploading Filename:${req.query?.filename}, Content type:${req.headers["content-type"]}, Length:${req.body.length}`

```

Comment:
do we want to include this ***?

Reply:
This has been copied directly from the azure function code sample in microsoft docs.
https://docs.microsoft.com/en-us/azure/developer/javascript/how-to/with-web-app/azure-function-file-upload